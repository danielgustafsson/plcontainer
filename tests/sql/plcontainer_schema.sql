
CREATE TABLE users (
	fname text not null,
	lname text not null,
	username text,
	userid serial
	) ;

CREATE TABLE taxonomy (
	id serial,
	name text
	) ;

CREATE TABLE entry (
	accession text not null,
	eid serial,
	txid int2 not null
	) ;

CREATE TABLE sequences (
	eid int4 not null,
	pid serial,
	product text not null,
	sequence text not null,
	multipart bool default 'false'
	) ;

CREATE TABLE xsequences (
	pid int4 not null,
	sequence text not null
	) ;

CREATE TABLE unicode_test (
	testvalue  text NOT NULL
);

CREATE TABLE table_record (
	first text,
	second int4
	) ;

CREATE TYPE type_record AS (
	first text,
	second int4
	) ;
