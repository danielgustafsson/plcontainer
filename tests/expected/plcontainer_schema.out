CREATE TABLE users (
	fname text not null,
	lname text not null,
	username text,
	userid serial
	) ;
NOTICE:  CREATE TABLE will create implicit sequence "users_userid_seq" for serial column "users.userid"
NOTICE:  Table doesn't have 'DISTRIBUTED BY' clause -- Using column named 'fname' as the Greenplum Database data distribution key for this table.
HINT:  The 'DISTRIBUTED BY' clause determines the distribution of data. Make sure column(s) chosen are the optimal data distribution key to minimize skew.
CREATE TABLE taxonomy (
	id serial,
	name text
	) ;
NOTICE:  CREATE TABLE will create implicit sequence "taxonomy_id_seq" for serial column "taxonomy.id"
NOTICE:  Table doesn't have 'DISTRIBUTED BY' clause -- Using column named 'id' as the Greenplum Database data distribution key for this table.
HINT:  The 'DISTRIBUTED BY' clause determines the distribution of data. Make sure column(s) chosen are the optimal data distribution key to minimize skew.
CREATE TABLE entry (
	accession text not null,
	eid serial,
	txid int2 not null
	) ;
NOTICE:  CREATE TABLE will create implicit sequence "entry_eid_seq" for serial column "entry.eid"
NOTICE:  Table doesn't have 'DISTRIBUTED BY' clause -- Using column named 'accession' as the Greenplum Database data distribution key for this table.
HINT:  The 'DISTRIBUTED BY' clause determines the distribution of data. Make sure column(s) chosen are the optimal data distribution key to minimize skew.
CREATE TABLE sequences (
	eid int4 not null,
	pid serial,
	product text not null,
	sequence text not null,
	multipart bool default 'false'
	) ;
NOTICE:  CREATE TABLE will create implicit sequence "sequences_pid_seq" for serial column "sequences.pid"
NOTICE:  Table doesn't have 'DISTRIBUTED BY' clause -- Using column named 'eid' as the Greenplum Database data distribution key for this table.
HINT:  The 'DISTRIBUTED BY' clause determines the distribution of data. Make sure column(s) chosen are the optimal data distribution key to minimize skew.
CREATE TABLE xsequences (
	pid int4 not null,
	sequence text not null
	) ;
NOTICE:  Table doesn't have 'DISTRIBUTED BY' clause -- Using column named 'pid' as the Greenplum Database data distribution key for this table.
HINT:  The 'DISTRIBUTED BY' clause determines the distribution of data. Make sure column(s) chosen are the optimal data distribution key to minimize skew.
CREATE TABLE unicode_test (
	testvalue  text NOT NULL
);
NOTICE:  Table doesn't have 'DISTRIBUTED BY' clause -- Using column named 'testvalue' as the Greenplum Database data distribution key for this table.
HINT:  The 'DISTRIBUTED BY' clause determines the distribution of data. Make sure column(s) chosen are the optimal data distribution key to minimize skew.
CREATE TABLE table_record (
	first text,
	second int4
	) ;
NOTICE:  Table doesn't have 'DISTRIBUTED BY' clause -- Using column named 'first' as the Greenplum Database data distribution key for this table.
HINT:  The 'DISTRIBUTED BY' clause determines the distribution of data. Make sure column(s) chosen are the optimal data distribution key to minimize skew.
CREATE TYPE type_record AS (
	first text,
	second int4
	) ;
